gnat2goto/driver/tree_walk.adb

```diff
@@ -377,6 +390,19 @@ package body Tree_Walk is
       Report_Unhandled_Node_Empty (N, Fun_Name, Message);
       return I_Empty;
    end Report_Unhandled_Node_Kind;
+
+   function Uint_To_Integer (Number : Uint) return Integer
+   is
+   begin
+      return Integer (UI_To_Int (Number));
```

Comment:
This restricts us to 32 bit bounds (`Int` is basically a 32 bit signed integer). We need to find a better way around this

Reply:
@hannes-steffenhagen-diffblue I've added one commit that addresses that.

Reply:
The commit is gone now, but basically we went from Int to Uint (which is of arbitrary size). It is now in separate package `range_check`.